towhee/runtime/data_queue.py

```diff
@@ -284,7 +301,7 @@ def __init__(self):
         self._data = None
 
     def put(self, data):
-        if isinstance(data, _Empty):
+        if data is Empty():
```

Comment:
This line checks if `data` and `Empty()` are the same object. Can we asume that all the instance of Empty are identical? 
I think we should avoid this kind of comparision and provide a util function like is_empty(value). Be careful about its implementation.

Reply:
Empty is a singleton, to avoid creating too many objs.